srcStatic/ModuleLoader.cpp

```diff
@@ -1,14 +1,30 @@
 #include "ModuleLoader.h"
+#include "GameModules/IpDropDown.h"
 #include "GameModules/IniModule.h"
 #include "StringConversion.h"
 #include "FileSystemHelper.h"
 
 
 ModuleLoader::ModuleLoader() 
 {
+	RegisterInternalModules();
 	RegisterExternalModules();
 }
 
+void ModuleLoader::RegisterInternalModules()
+{
+	const auto internalModuleNames = GetModuleNames("InternalModules");
+
+	for (const auto& moduleName : internalModuleNames) {
+		if ("ipdropdown" == ToLower(moduleName)) {
+			RegisterModule(static_cast<std::unique_ptr<GameModule>>(std::make_unique<IPDropDown>()));
```

Comment:
The boilerplate `if` statements needed here will likely grow quite substantially as more modules are added. I suspect there is an easier way to handle this with templates, and probably with a tuple listing all the built in module classes.